dds_web/__init__.py

```diff
@@ -6,6 +6,7 @@
 
 # Standard library
 import logging
+import datetime
```

Comment:
minor thing, but maybe sort them alphabetically?

Reply:
Agreed, but this is either a separate task for (much) later, or during other prioritized tasks 👍🏻